.#test_wrappers.py

```
@@ -0,0 +1 @@
+gtk@gtkoffice.3337573:1736749975
```

Ruff (0.8.2)

No such file or directory (os error 2)

(E902)